src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -9,14 +9,6 @@
   list-style: none;
 }
 
-.homepage-hub, .va-sidebarnav, .vet-toolbar {
-  [href^="http"]:not(.usa-button-primary), [rel="external"] {
-    background-image: none;
-    padding-right: 0px;
-
-  }
-}
-
```

Comment:
This was an override to remove the external link icon, but that was already done in the design system here, https://github.com/department-of-veterans-affairs/design-system/pull/198.